notimetowait2.md

```diff
@@ -41,10 +41,13 @@ The event will also present on the current state of the IETF’s [CELLAR](https:
 ## When
 Thursday, November 9, 2017 and Friday, November 10, 2017
 
+Opening: 09:00
+Closing: 17:00 (sharp)
+
 ## Where
 [Österreichisches Filmmuseum - Austrian Film Museum](https://www.filmmuseum.at)
 
-![Österreichisches Filmmuseum - Austrian Film Museum](/MediaConch/images/Filmmuseum_04s.jpeg)  
+![Österreichisches Filmmuseum - Austrian Film Museum](/MediaConch/images/Filmmuseum_04s.jpeg)
```

Comment:
This double space at the end prevents the next line from wrapping into this one.

Reply:
+1